### PR TITLE
Add some components that used in asyncy.com

### PR DESCRIPTION
**Pull request by tiny-tinker**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.story.js
+/src/stories/*
```
Comment (JeanBarriere) [inline .npmignore]: can be improved i guess (all the config files are not required also, same for the tests files) do you agree ?

File: src/stories/components/Nav/Nav.story.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { storiesOf } from '@storybook/vue'
+import logo from '../../../assets/images/logo.svg'
```
Comment (JeanBarriere) [inline src/stories/components/Nav/Nav.story.js]: better usign alias `@/assets/` instead of relative path `../../` here

Comment (tiny-tinker) [reply src/stories/components/Nav/Nav.story.js]: Yep. add the @assets alias in webpack config. :)

File: .npmignore (hunk 1 above)
Comment (JeanBarriere) [reply .npmignore]: don't think so, sounds good

**Pull request merged**